patches/PR1/linkoptforzos.patch

```diff
@@ -14,6 +14,15 @@ index 2fbefbc..ddbe4a3 100644
  else
  	SONAME_FLAGS = -Wl,-soname=libxxhash.$(SHARED_EXT).$(LIBVER_MAJOR)
  	SHARED_EXT = so
+@@ -99,7 +104,7 @@ xxhsum: CPPFLAGS += -DXXHSUM_DISPATCH=1
+ xxhsum: xxh_x86dispatch.o
+ endif
+ xxhsum: xxhash.o $(XXHSUM_SPLIT_OBJS)
+-	$(CC) $(FLAGS) $^ $(LDFLAGS) -o $@$(EXT)
++	$(CC) $(FLAGS) $^ $(ZOPEN_EXTRA_CPPFLAGS) $(LDFLAGS) $(ZOPEN_EXTRA_LIBS) $(ZOPEN_EXTRA_LDFLAGS) -o $@$(EXT)
```

Comment:
Can we use CPPFLAGS and LIBS and LDFLAGS instead to make it zopen agnostic?

Reply:
Sure, done.